README.md

```diff
@@ -383,6 +383,76 @@ using (var command = new TraceableSqlCommand("SELECT * FROM products", connectio
 2. Parameterized values will appear in their tokenized form and will not be expanded.
 3. The value of `collectSqlQueries` in the `TraceableSqlCommand` instance overrides the value set in the global configuration using the `CollectSqlQueries` property.
 
+### Trace SQL Query through Entity Framework Core 3.0 and above (.NET Core)
+
+AWS XRay SDK for .NET Core provides interceptor for tracing SQL query through Entity Framework Core (>=3.0).
+
+For how to start with Entity Framework Core in an ASP.NET Core web app, please take reference to [Link](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/intro?view=aspnetcore-3.1&tabs=visual-studio)
+
+*NOTE:*
+
+* Not all database provider support Entity Framework Core 3.0 and above, please make sure that you are using the [Nuget package](https://docs.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli) with a compatible version (EF Core >= 3.0).
+
+#### Setup
+
+In order to trace SQL query, you can register your `DbContext` with `EFInterceptor` in the `ConfigureServices` method in `startup.cs` file. 
+
+For instance, when dealing with MySql server using Nuget: [Pomelo.EntityFrameworkCore.MySql](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql) (V 3.1.1). 
+
+```csharp
+public void ConfigureServices(IServiceCollection services){ 
+
+    services.AddDbContext<your_DbContext>(options => options.UseMySql(your_connectionString).AddInterceptors(new EFInterceptor()));
```

Comment:
To improve discover ability I would suggest adding an extension method to `DbContextOptionsBuilder` called `AddXRayTracing` or something which do the work of calling  `AddInterceptors` with the `EFInterceptor`. That way Intellisense would pretty tell the user how to enable the feature instead of relying on the developer to understand about interceptors and what class that should create and pass in. This approach is similar to the `UseXRay` extension method used in the ASP.NET Core handler.